modules/wmco-prerequisites.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/understanding-windows-container-workloads.adoc
+
+[id="wmco-prerequisites_{context}"]
+= Windows Machine Config Operator prerequisites
+
+The following information details the supported cloud provider versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
+
+== Supported cloud providers based on {product-title} and WMCO versions
+
+[cols="3",options="header"]
+|===
+|Cloud provider
+|Supported {product-title} version
+|Supported WMCO version
+
+|Amazon Web Services (AWS)
+|4.6+
+|WMCO 1.0+
+
+|Microsoft Azure
+|4.6+
+|WMCO 1.0+
+
+|VMware vSphere
+|4.7+
+|WMCO 2.0+
+|===
+
+## Supported Windows Server versions
```

Comment:
Despite that Markdown-style heading seem to work, please revise to AsciiDoc flavor like "== Supported...".

Apologies for missing earlier.